### PR TITLE
Update records-management

### PR DESCRIPTION
**Pull request by Sherylt**
V1.0 of new Records Management page

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/records-management [hunk 1]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-  
-  - A system for monitoring and reporting on the handling of records to ensure that employees are filing, accessing, and managing them according to defined policies and processes.
-  
-  - A process and the data to support an auditable and defensible content and records lifecycle to provide to internal or external audit, legal and/or regulatory authorities for review. 
-
-
-## Benefits
-
-Lorem ....
+  - **Content analysis** that identifies and categorizes content in the enterprise that: are records, work in progress, or have business general use value. The analysis will also provide information about where the content resides, which business functions are custodians of the content, and how the content needs to be handled throughout its lifecycle.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
  - **Content analysis** that identifies and categorizes content in the enterprise that are records, work in progress, or have business general use value. The analysis will also provide information about where the content resides, which business functions are custodians of the content, and how the content needs to be handled throughout its life cycle.
```

File: SecurityCompliance/records-management [hunk 2]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-    o	Via Sensitive information
-    o	Via content query
-    o	Via default label at library, folder and docset levels
-    o	When an event occurs – project closure, employee leaves
-•	Compliant and defensible disposition
-    o	Manual Review content for disposal 
-    o	Automate disposal for non-value content
-•	Visually monitor retention, records and disposition activity
-•	Quick access to audit and activity reports
+The key components of records management in Office 365 are provided by the Security and Compliance Center are:
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
The key components of records management in Office 365 provided by the Security and Compliance Center are: 
```

File: SecurityCompliance/records-management [hunk 3]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-•	Import existing file plan
-•	Ensure compliance for regulatory records
-•	Apply labels by default to a library, folder or docset
-•	Auto-apply labels based on your existing SharePoint metadata
-•	Auto-apply labels based on sensitive content 
-•	Add a site collection to an existing retention label policy
-•	How to handle content left behind by an employee no longer with the organization
+Through the use of these components, your organization will minimally realize the following benefits: 
+
+- A centralized administrative portal, specific for Records and Compliance Managers
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- A centralized administrative portal, specific for records and compliance managers 
```

File: SecurityCompliance/records-management [hunk 4]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-•	Ensure compliance for regulatory records
-•	Apply labels by default to a library, folder or docset
-•	Auto-apply labels based on your existing SharePoint metadata
-•	Auto-apply labels based on sensitive content 
-•	Add a site collection to an existing retention label policy
-•	How to handle content left behind by an employee no longer with the organization
+Through the use of these components, your organization will minimally realize the following benefits: 
+
+- A centralized administrative portal, specific for Records and Compliance Managers 
+- Supports various combinations of retention, record declaration snd disposition rules
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- Support for various combinations of retention, record declaration, and disposition rules
```

File: SecurityCompliance/records-management [hunk 5]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- In-place Records Management: no need to move contnet
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- In-place records management; no need to move content
```

File: SecurityCompliance/records-management [hunk 6]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- Cross-O365 coverage: Exchange, SharePoint, OneDrive, Groups, Teams
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- Cross-O365 coverage; Exchange, SharePoint, OneDrive, Groups, Teams
```

File: SecurityCompliance/records-management [hunk 7]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- The ability to present the most relevant, in context retention labels to users
+- Automatically apply retention label, based on: sensitive information, content query, default retention label
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- Automatically apply retention label based on sensitive information, content query, and default retention label
```

File: SecurityCompliance/records-management [hunk 8]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- Declare records within business process workflpply a retention label when an event occurs – project closure, employee leaves
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- Declare records within business process workflow. Apply a retention label when an event occurs – project closure, employee leaves
```

File: SecurityCompliance/records-management [hunk 9]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- Compliant and defensible disposition
+   - Manually review content for disposal 
+   - Automate disposal for non-value content
+- Visually monitor retention, records and disposition activity
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
- Visually monitor retention, records, and disposition activity
```

File: SecurityCompliance/records-management [hunk 10]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+- Quick access to activity reports
+
+## Business scenarios
+
+Here are a few hightlights of the many business scenarios supported: 
+
+  - Import your existing file plan
+  - Ensure regulatory compliance for records
+  - Apply labels by default to a library, folder or docset
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
  - Apply labels by default to a library, folder, or docset
```

File: SecurityCompliance/records-management [hunk 11]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+  - Auto-apply labels based on your existing SharePoint metadata
+  - Auto-apply labels based on sensitive content, like social security numbers 
+  - Add a site collection to an existing retention label policy
+  - How to handle content left behind by an employee no longer with the organization
+  
+## File plan
 
+**File plan management**
+Centralized and advanced management of the entire content lifecycle - spanning retention labels, retention label policies, events, record declaration, disposition review, analytics and reporting.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
Centralized and advanced management of the entire content lifecycle - spanning retention labels, retention label policies, events, record declaration, disposition review, analytics, and reporting. 
```

File: SecurityCompliance/records-management [hunk 12]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+  - How to handle content left behind by an employee no longer with the organization
+  
+## File plan
 
+**File plan management**
+Centralized and advanced management of the entire content lifecycle - spanning retention labels, retention label policies, events, record declaration, disposition review, analytics and reporting. 
 
-## Step-by-step
+**File plan descriptors**
+Use descriptors to organize and differentiate labels from each other – Reference ID, Function/department, Category, Subcategory, Authority type and Provision/citation.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
Use descriptors to organize and differentiate labels from each other – Reference ID, Function/department, Category, Subcategory, Authority type, and Provision/citation.
```

File: SecurityCompliance/records-management [hunk 13]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-  •	Content managers to find where organizational information is kept and to make sure that that their teams follow records management practices.
+**Active/Inactive Status**
+Know which retention labels and retention label policies are actively in use and which aren’t by referencing the status of each.
 
-### Step 3: Create your organization’s file plan
-•	Records retention schedule 
-•	Maybe there are retention labels and policies already in O365
-•	Manually create/update or import
-•	…
+[Learn more about the capabilities, and benefits of file plan.](file-plan-manager.md)
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
[Learn more about the capabilities and benefits of file plan.](file-plan-manager.md) 
```

File: SecurityCompliance/records-management [hunk 14]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+[Learn more about the capabilities, and benefits of file plan.](file-plan-manager.md) 
 
-### Step 4: Apply retention labels to content
- •	Automate and/or manually apply
- •	Management of obsolete records
+## Events
 
-### Step 5: Monitor & fine tune your organization’s file plan
- •	Man
+[Learn more about the capabilities, and benefits of Events.](event-driven-retention.md)
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
[Learn more about the capabilities and benefits of Events.](event-driven-retention.md)
```

File: SecurityCompliance/records-management [hunk 15]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
 
-### Step 6: Review content for disposition
+## Dispositions
 
-### Step 7: Provide data as required/requested by governing bodies
-•	Defensible deletion
-•	…
+[Learn more about the capabilities, and benefits of disposition reviews.](disposition-reviews.md)
```
Comment (JohanFreelancer9) [inline SecurityCompliance/records-management]:
```suggestion
[Learn more about the capabilities and benefits of disposition reviews.](disposition-reviews.md)
```

File: SecurityCompliance/records-management [hunk 16]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
 ---
 # Records management in Office 365
 
-Records management is all about keeping data around when you need it and getting rid of it when you don't. A well-designed records management program helps protect an organization legally, provides the ability to demonstrate compliance with regulatory obligations, and increases efficiency by promoting the disposition items that are no longer of value or no longer required to be retained by the organization. 
-
-With records management in Office 365, you can apply your organization retention schedules into the file plan to manage retention, records declaration and disposition in support of the full content lifecycle and accross Office 365.
-
-## Overview
+Records management is all about keeping data around when you need it and getting rid of it when you don't. A well-designed records management program helps protect an organization legally, provides the ability to demonstrate compliance with regulatory obligations, and increases efficiency by promoting the disposition of  items that are no longer of value or no longer required to be retained by the organization.
```
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
Records management is all about keeping data around when you need it and getting rid of it when you don't. A well-designed records management program helps protect an organization legally, provides the ability to demonstrate compliance with regulatory obligations, and increases efficiency by promoting the disposition of items that are no longer of value, or no longer required by the organization. 
```

File: SecurityCompliance/records-management [hunk 17]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
 ---
 # Records management in Office 365
 
-Records management is all about keeping data around when you need it and getting rid of it when you don't. A well-designed records management program helps protect an organization legally, provides the ability to demonstrate compliance with regulatory obligations, and increases efficiency by promoting the disposition items that are no longer of value or no longer required to be retained by the organization. 
-
-With records management in Office 365, you can apply your organization retention schedules into the file plan to manage retention, records declaration and disposition in support of the full content lifecycle and accross Office 365.
-
-## Overview
```
Comment (nenonix) [inline SecurityCompliance/records-management]: A level 2 heading here would help to break up the text and make it more scannable. Suggest you retain ## Overview where it was

File: SecurityCompliance/records-management [hunk 18]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
-  
-  - A process and the data to support an auditable and defensible content and records lifecycle to provide to internal or external audit, legal and/or regulatory authorities for review. 
-
-
-## Benefits
-
-Lorem ....
+  - **Content analysis** that identifies and categorizes content in the enterprise that: are records, work in progress, or have business general use value. The analysis will also provide information about where the content resides, which business functions are custodians of the content, and how the content needs to be handled throughout its lifecycle.
+  - **Compliance requirements** that identify which regulations and/or laws are applicable, articulate the rules that the organization must follow to ensure compliance, and the methods that need to be used to ensure the participation of enterprise team members.  
+  - **Operational and human resources requirements** that identify the authorities, stakeholders and accountable parties within the organization that are required to act for the organization to be successful in complying with applicable laws/regulations and to meet the business continuity goals of the records management program.
```
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
  - **Operational and human resources requirements** that identify the authorities, stakeholders and accountable parties within the organization that are required to act for the organization to be successful in complying with applicable laws/regulations, and to meet the business continuity goals of the records management program.
```

File: SecurityCompliance/records-management [hunk 19]
```diff
@@ -5,108 +5,89 @@ description: "With records management in Office 365, you can apply your organiza
...
+
+Here are a few hightlights of the many business scenarios supported: 
+
+  - Import your existing file plan
+  - Ensure regulatory compliance for records
+  - Apply labels by default to a library, folder or docset
+  - Auto-apply labels based on your existing SharePoint metadata
+  - Auto-apply labels based on sensitive content, like social security numbers 
+  - Add a site collection to an existing retention label policy
+  - How to handle content left behind by an employee no longer with the organization
```
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
  - Handle content left behind by an employee no longer with the organization
```

File: SecurityCompliance/records-management (hunk 15 above)
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
Learn more about the capabilities and benefits of [disposition reviews](disposition-reviews.md).
```

File: SecurityCompliance/records-management (hunk 13 above)
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
Learn more about the capabilities and benefits of [file plan](file-plan-manager.md).
```

File: SecurityCompliance/records-management (hunk 14 above)
Comment (nenonix) [inline SecurityCompliance/records-management]:
```suggestion
Learn more about the capabilities and benefits of [Events Driven Retention](event-driven-retention.md).
```

**Comment by stephow-MSFT** [comment]
We published a different version of this file via the private repo, so not merging this one.

**Pull request closed without merging**